tests/core/mountpoint/test_base.py

```diff
@@ -275,8 +275,10 @@ async def test_get_path_in_mountpoint(base_mountpoint, alice_user_fs, event_bus)
         bar_path = mountpoint_manager.get_path_in_mountpoint(wid, FsPath("/bar.txt"))
 
         assert isinstance(bar_path, PurePath)
-        expected = base_mountpoint / f"mounted_wksp" / "bar.txt"
-        assert str(bar_path) == str(expected.absolute())
+        # Windows uses drives, not base_mountpoint
+        if os.name != "nt":
+            expected = base_mountpoint / f"mounted_wksp" / "bar.txt"
```

Comment:
no need for the fstring, and a single string would save one Path construction ^^

Reply:
:+1: